tpl/tplimpl/template_embedded.go

```diff
@@ -84,7 +84,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      <description>{{ .Content | html }}</description>
+      <description>{{ .Summary | html }}</description>
```

Comment:
@DarwinJS I'm at a loss on that one man, haha. I don't actually use MailChimp, but FWIW, I don't think it should be up to mailchimp to redefine the spec. Let's see what @digitalcraftsman says about it and go from there. BTW, thanks for submitting this PR; I hadn't ever looked at the built-in template.